src/daisy_seed.cpp

```diff
@@ -1,4 +1,5 @@
 #include "daisy_seed.h"
+#include "stm32h7xx_hal.h"
```

Comment:
Do you still need this for something?

If not, lets get rid of that, otherwise let's update it to the hal_map file that way there's a single include for any libDaisy sources that need access to the HAL (I still dream of a HAL-less future).